package.json

```diff
@@ -74,6 +75,7 @@
     "qrcode": "^1.5.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-toastify": "^9.1.3",
```

Comment:
Since we will likely we switching to using shadcn ui components in the near future (see #191), may be we can use https://ui.shadcn.com/docs/components/toast component here.